backends/xnnpack/runtime/XNNCompiler.cpp

```diff
@@ -630,7 +630,14 @@ Error defineConvertNode(
       subgraph_ptr,
       remapped_ids.at(graph_node->input_id()),
       remapped_ids.at(graph_node->output_id()),
+#ifdef ENABLE_XNNPACK_KLEIDI
+      // This maps to XNNPACK's XNN_FLAG_MAYBE_PACK_FOR_QB4W_GEMM
```

Comment:
perhaps we should fix it in XNNPACK, independent of this PR

Reply:
I don't think they want this public yet, since it would cause backwards compatibility issues in the future. This is likely the path until they have a complete qp8 story.